src/eFAST.jl

```diff
@@ -0,0 +1,65 @@
+Base.@kwdef struct eFAST
+    num_harmonics::Int=4
```

Comment:
what's this parameter?

Reply:
The number of harmonics to sum in the Fourier series decomposition